app.go

```diff
@@ -155,6 +158,59 @@ func getApp(client *cmd.Client, appName string) (app, error) {
 	return a, err
 }
 
+func setEnvs(client *cmd.Client, appName string, envVars *api.Envs) error {
+	url, err := cmd.GetURL("/apps/" + appName + "/env")
+	if err != nil {
+		return err
+	}
+	v, err := form.EncodeToValues(envVars)
+	if err != nil {
+		return err
+	}
+	req, err := http.NewRequest("POST", url, strings.NewReader(v.Encode()))
+	if err != nil {
+		return err
+	}
+	req.Header.Set("Content-Type", "application/x-www-form-urlencoded")
+	resp, err := client.Do(req)
+	if err != nil {
+		return err
+	}
+	return cmd.StreamJSONResponse(ioutil.Discard, resp)
+}
+
+func getEnvs(client *cmd.Client, appName string) ([]envVar, error) {
+	var vars []envVar
+	resp, err := doReq(client, "/apps/"+appName+"/env")
```

Comment:
@Lepthy our code currently passes gofmt checking (i.e. if I change this line or app.go:162, the build will break). It looks like when there's a single parameter, gofmt enforces spaces around operators, but with multiple params, it removes the space: https://play.golang.org/p/IGIoRYJYYV (click "Format").

I have no idea why :-D